swarm.cabal

```diff
@@ -295,7 +625,7 @@ test-suite swarm-integration
 benchmark benchmark
   import:         stan-config, common, ghc2021-extensions
   main-is:        Benchmark.hs
-  hs-source-dirs: bench
+  hs-source-dirs: test/bench
```

Comment:
@byorgey I moved the `app` and `bench` directories to `src` and `test` respectively.

Sometimes I forgot to recompile the benchmark until the CI told me, so I hope seeing it with other tests will remind me about it. :slightly_smiling_face: